src/OSPSuite.Presentation/Binders/ObservedDataDragDropBinder.cs

```diff
@@ -24,6 +25,26 @@ public IReadOnlyList<DataRepository> DroppedObservedDataFrom(IDragEvent e)
          return getObservedDataNodesFrom(e).Select(observedDataNode => observedDataNode.Tag.Subject).ToList();
       }
 
+      public List<List<DataRepository>> DroppedObservedDataWithFolderPathFrom(IDragEvent e)
```

Comment:
@georgeDaskalakis I am still confused as to where this is used?DroppedObservedDataWithFolderPathFrom
Why do you need List? IReadOnly is probably even better from an API point of view